pennylane/ftqc/decomposition.py

```diff
@@ -43,3 +52,232 @@ def convert_to_mbqc_gateset(tape):
         )
     tapes, fn = decompose(tape, gate_set=mbqc_gate_set, alt_decomps={Rot: [_rot_to_xzx]})
     return tapes, fn
+
+
+@transform
+def convert_to_mbqc_formalism(tape):
+    if len(tape.measurements) != 1 or not isinstance(tape.measurements[0], SampleMP):
```

Comment:
counts should also be valid

Reply:
On second thought, while this is technically true for this transform, in general for FTQC we need `samples` so we can resolve and correct issues with single-shot measurements, so not really useful. Sticking to `sample` only.